react-list.es6

```diff
@@ -126,6 +136,7 @@ module.exports = class ReactList extends Component {
     window.removeEventListener('resize', this.updateFrame);
     this.scrollParent.removeEventListener('scroll', this.updateFrame, PASSIVE);
     this.scrollParent.removeEventListener('mousewheel', NOOP, PASSIVE);
+    clearTimeout(this.updateFrameTimeoutId);
```

Comment:
Let's check if `this.updateFrameTimeoutId` exists first. I believe some JS interpreters throw when trying to `clearTimeout` on `undefined`.